src/System.Runtime.Serialization.Formatters/tests/FormatterServicesTests.cs

```diff
@@ -109,13 +109,6 @@ public static IEnumerable<object[]> GetUninitializedObject_ByRefLikeType_TestDat
             if (!PlatformDetection.IsNetNative)
```

Comment:
You can delete the `if (!PlatformDetection.IsNetNative)` condition and the obsolete comment in front of it too while you are on it.

The test for `RuntimeArgumentHandle` should work just fine even on .NET Native.